test/cpp_extensions/open_registration_extension/torch_openreg/csrc/CMakeLists.txt

```diff
@@ -9,4 +9,16 @@ add_library(${LIBRARY_NAME} SHARED ${SOURCE_FILES})
 target_link_libraries(${LIBRARY_NAME} PRIVATE torch_cpu_library openreg)
 target_include_directories(${LIBRARY_NAME} PUBLIC ${CMAKE_CURRENT_SOURCE_DIR})
 
-install(TARGETS ${LIBRARY_NAME} LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR})
+if (WIN32)
+    target_compile_definitions(${LIBRARY_NAME} PRIVATE OPENREG_EXPORTS)
+endif()
+
+if(WIN32)
+    install(TARGETS ${LIBRARY_NAME} 
+        ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
+        LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
+        RUNTIME DESTINATION ${CMAKE_INSTALL_LIBDIR}
+    )
+else()
+    install(TARGETS ${LIBRARY_NAME} LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR})
+endif()
```

Comment:
Please delete all of this code and keep only the following so that all platforms are consistent.

```CMake
install(TARGETS ${LIBRARY_NAME}
Archive target ${CMAKE_INSTALL_LIBDIR}
Library target ${CMAKE_INSTALL_LIBDIR}
Runtime target ${CMAKE_INSTALL_LIBDIR}
)
```